host: nid001005
Building DAG of jobs...
You are running snakemake in a SLURM job context. This is not recommended, as it may lead to unexpected behavior. Please run Snakemake directly on the login node.
SLURM run ID: 84ef4593-1287-483e-aa88-7d743461b7ea
Using shell: /usr/local/bin/bash
Provided remote nodes: 10
Job stats:
job                                 count
--------------------------------  -------
create_inference_pyproject              3
create_inference_venv                   3
experiment_all                          1
inference_group_forecaster              1
inference_group_interpolator            2
make_squashfs_image                     3
map_init_time_to_inference_group        8
report_experiment_dashboard             1
verif_metrics                           8
verif_metrics_aggregation               2
verif_metrics_aggregation_cosmoe        1
verif_metrics_cosmoe                    2
verif_metrics_plot                      1
total                                  36

Select jobs to execute...
Execute 5 jobs...

[Wed Aug 20 13:23:34 2025]
localrule create_inference_pyproject:
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/d0846032fc7248a58b089cbe8fa4c511/pyproject.toml
    log: output/logs/create_inference_pyproject/d0846032fc7248a58b089cbe8fa4c511.log
    jobid: 13
    reason: Missing output files: output/data/runs/d0846032fc7248a58b089cbe8fa4c511/pyproject.toml
    wildcards: run_id=d0846032fc7248a58b089cbe8fa4c511
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/tmp, slurm_partition=postproc, cpus_per_task=1, mem_mb_per_cpu=1800, runtime=60
[Wed Aug 20 13:23:34 2025]
localrule verif_metrics_cosmoe:
    input: workflow/scripts/verif_cosmoe_fct.py, /store_new/mch/msopr/ml/COSMO-E/FCST20.zarr, /scratch/mch/fzanetta/data/anemoi/datasets/mch-co2-an-archive-0p02-2015-2020-6h-v3-pl13.zarr
    output: output/data/baselines/COSMO-E/202001011200/verif.csv
    log: output/logs/verif_metrics_cosmoe/202001011200.log
    jobid: 3
    reason: Missing output files: output/data/baselines/COSMO-E/202001011200/verif.csv
    wildcards: init_time=202001011200
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/tmp, slurm_partition=postproc, cpus_per_task=1, mem_mb_per_cpu=1800, runtime=60
[Wed Aug 20 13:23:34 2025]
localrule create_inference_pyproject:
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/07c3d9698db14d859b78bb712a65bbbf/pyproject.toml
    log: output/logs/create_inference_pyproject/07c3d9698db14d859b78bb712a65bbbf.log
    jobid: 26
    reason: Missing output files: output/data/runs/07c3d9698db14d859b78bb712a65bbbf/pyproject.toml
    wildcards: run_id=07c3d9698db14d859b78bb712a65bbbf
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/tmp, slurm_partition=postproc, cpus_per_task=1, mem_mb_per_cpu=1800, runtime=60
[Wed Aug 20 13:23:34 2025]
localrule create_inference_pyproject:
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/9c18b90074214d769b8b383722fc5a06/pyproject.toml
    log: output/logs/create_inference_pyproject/9c18b90074214d769b8b383722fc5a06.log
    jobid: 9
    reason: Missing output files: output/data/runs/9c18b90074214d769b8b383722fc5a06/pyproject.toml
    wildcards: run_id=9c18b90074214d769b8b383722fc5a06
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/tmp, slurm_partition=postproc, cpus_per_task=1, mem_mb_per_cpu=1800, runtime=60
[Wed Aug 20 13:23:34 2025]
localrule verif_metrics_cosmoe:
    input: workflow/scripts/verif_cosmoe_fct.py, /store_new/mch/msopr/ml/COSMO-E/FCST20.zarr, /scratch/mch/fzanetta/data/anemoi/datasets/mch-co2-an-archive-0p02-2015-2020-6h-v3-pl13.zarr
    output: output/data/baselines/COSMO-E/202001060000/verif.csv
    log: output/logs/verif_metrics_cosmoe/202001060000.log
    jobid: 4
    reason: Missing output files: output/data/baselines/COSMO-E/202001060000/verif.csv
    wildcards: init_time=202001060000
    resources: mem_mb=1000, mem_mib=954, disk_mb=1000, disk_mib=954, tmpdir=/tmp, slurm_partition=postproc, cpus_per_task=1, mem_mb_per_cpu=1800, runtime=60
Traceback (most recent call last):
  File "/users/omiralle/evalml/.snakemake/scripts/tmpj9wr51n9.set_inference_pyproject.py", line 435, in <module>
    raise SystemExit(main(snakemake))
                     ^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.snakemake/scripts/tmpj9wr51n9.set_inference_pyproject.py", line 393, in main
    client = get_mlflow_client_given_runid(mlflow_uri, run_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.snakemake/scripts/tmpj9wr51n9.set_inference_pyproject.py", line 100, in get_mlflow_client_given_runid
    client.get_run(run_id)
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/tracking/client.py", line 275, in get_run
    return self._tracking_client.get_run(run_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/tracking/_tracking_service/client.py", line 100, in get_run
    return self.store.get_run(run_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/store/tracking/rest_store.py", line 228, in get_run
    response_proto = self._call_endpoint(GetRun, req_body)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/store/tracking/rest_store.py", line 134, in _call_endpoint
    return call_endpoint(
           ^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/utils/rest_utils.py", line 554, in call_endpoint
    response = verify_rest_response(response, endpoint)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.venv/lib/python3.12/site-packages/mlflow/utils/rest_utils.py", line 326, in verify_rest_response
    raise MlflowException(f"{base_msg}. Response body: '{response.text}'")
mlflow.exceptions.MlflowException: API request to endpoint was successful but the response body was not in a valid JSON format. Response body: '<!DOCTYPE html>
<html class="login-pf">

<head>
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta name="robots" content="noindex, nofollow">

            <meta name="viewport" content="width=device-width,initial-scale=1"/>
    <title>Log in to ECMWF</title>
    <link rel="icon" href="/auth/resources/m5dds/login/ECMWF-THEME/img/favicon.ico" />
            <link href="/auth/resources/m5dds/common/keycloak/web_modules/@patternfly/react-core/dist/styles/base.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/common/keycloak/web_modules/@patternfly/react-core/dist/styles/app.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/common/keycloak/node_modules/patternfly/dist/css/patternfly.min.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/common/keycloak/node_modules/patternfly/dist/css/patternfly-additions.min.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/common/keycloak/lib/pficon/pficon.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/login/ECMWF-THEME/css/login.css" rel="stylesheet" />
            <link href="/auth/resources/m5dds/login/ECMWF-THEME/css/tile.css" rel="stylesheet" />
</head>

<body class="">

   <!-- Google Tag Manager (noscript) -->
   <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-P5T597" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
   <!-- End Google Tag Manager (noscript) -->



 <!-- Include the custom header -->


<div class="content-wrapper">
<div class="custom-header">
    <div class="header-content">
        <!-- Logo on the left -->
        <a href="https://www.ecmwf.int" class="logo">
            <img src="/auth/resources/m5dds/login/ECMWF-THEME/img/logo.png" alt="ECMWF">
        </a>

        <!-- Right side items -->
        <div class="header-links">
            <!-- Help Link with "?" icon -->
            <a href="https://www.ecmwf.int/en/support" class="help-link">
                &#9432; Help
            </a>

            
        </div>
    </div>
</div>


 <div class="left-column">

   <div class="login-pf-page">
    <div class="card-pf">


        <header class="login-pf-header">
                <h1 id="kc-page-title">


</h1>
      </header>


      <div id="kc-content">
        <div id="kc-content-wrapper">




    <h1> Log in to your account</h1>
    <div id="kc-form">
     <div id="kc-form-wrapper">

            <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="https://accounts.ecmwf.int/auth/realms/ecmwf/login-actions/authenticate?session_code=luoiQmCYtVMfqEWc0Dab6WVQ0TCXx1IcgQi12khu4w4&amp;execution=408a29c4-9451-4b7e-b750-a4935f05abd0&amp;client_id=mlflow&amp;tab_id=5BwGBdQMHiY" method="post">

                    <div class="form-group">
                        <label for="username" class="pf-c-form__label pf-c-form__label-text">
                            Username or Email
                        </label>

                        <input tabindex="1" id="username" class="pf-c-form-control" name="username" value=""  
                               type="text" autofocus autocomplete="off" aria-invalid=""
                        />


                    </div>

                <div class="form-group">
                    <label for="password" class="pf-c-form__label pf-c-form__label-text">Password</label>

                    <input tabindex="2" id="password" class="pf-c-form-control" name="password" type="password" autocomplete="off"
                           aria-invalid=""
                    />


                </div>

                  <div id="kc-form-buttons" class="form-group">
                      <input type="hidden" id="id-hidden-input" name="credentialId" />
                      <input tabindex="4" class="pf-c-button pf-m-primary pf-m-block btn-lg" name="login" id="kc-login" type="submit" value="Log In"/>
                  </div>


                <div class="form-group login-pf-settings">
                    <div id="kc-form-options">
                        <div class="">
                                <span><a tabindex="5" href="/auth/realms/ecmwf/login-actions/reset-credentials?client_id=mlflow&amp;tab_id=5BwGBdQMHiY">Forgot Password?</a></span>
                        </div>
                    </div>
                </div>



            </form>
        
     </div>
    </div>









              <div id="kc-info" class="login-pf-signup">
                  <div id="kc-info-wrapper" class="">



                  </div>
              </div>

        </div>
      </div>
    </div>
   </div>
 </div>


  <!-- Right Column: New User Info -->
  <div class="right-column">
<div class="user-info-box">
    <h1>An ECMWF account enables you to:</h1>
                        <ul>
                                <li> access open data more quickly</li>
                                <li> register for events</li>
                                <li> enrol on online courses</li>
                                <li> access training resources</li>
                                <li> create and track service requests</li>
                        </ul>
   <p>To check if you are eligible for more features you can read about <a href="https://www.ecmwf.int/en/forecasts/accessing-forecasts">access to forecast data</a> and <a href="https://www.ecmwf.int/en/computing/access-computing-facilities">access to our computing facilities</a>.</p>
   <div id="kc-form-buttons" class="form-group">
        <a href="/auth/realms/ecmwf/login-actions/registration?client_id=mlflow&amp;tab_id=5BwGBdQMHiY" class="pf-c-button pf-m-primary pf-m-block btn-lg">Register new user</a>
   </div>


        

</div>

  </div>



</div>


<!-- Include the custom footer -->
<div class="custom-footer container-fluid">

        <div class="region region-footer">
            <div class="col-left text-left">
                <p class="copyright">© European Centre for Medium-Range Weather Forecasts</p>
            </div>
            <div class="col-right text-left">
                <ul class="legal-menu">
                  <li><a href="/en/accessibility" title="Useful footer links and privacy" data-drupal-link-system-path="node/20981">Accessibility</a></li>
                  <li><a href="/en/privacy" data-drupal-link-system-path="node/20982">Privacy</a></li>
                  <li><a href="/en/terms-use" data-drupal-link-system-path="node/20983">Terms of use</a></li>
                  <li><a href="/en/about/contact-us" data-drupal-link-system-path="node/20980">Contact</a></li>
                </ul>
            </div>
        </div>
    
</div>




</body>
</html>


<style>
/* Style for username and password fields */
input[name="username"], input[name="password"] {
    font-size: 24px;
    padding: 12px;
    border: 2px solid #007bff;
    border-radius: 5px;
    background-color: #ffffff;
    color: #000000;
    width: 100%;
    box-shadow: 0px 0px 10px rgba(0, 123, 255, 0.5);
    height: 3.5rem;
}

/* Input fields focus effect */
input[name="username"]:focus, input[name="password"]:focus {
    border-color: #ff9800;
    outline: none;
    box-shadow: 0px 0px 15px rgba(255, 152, 0, 0.7);
}

/* Labels styling */
label {
    font-size: 16px;
    font-weight: bold;
    color: #333;
    display: block;
    margin-bottom: 5px;
}

/* Submit button */
input[type="submit"] {
    font-size: 24px;
    padding: 12px;
    background-color: #28a745;
    color: white;
    border: none;
    border-radius: 5px;
    cursor: pointer;
    width: 100%;
    height: 3.5rem;
    transition: 0.3s;
}

input[type="submit"]:hover {
    background-color: #218838;
}



/* Responsive Design */
@media (max-width: 768px) {
    .content-wrapper {
        flex-direction: column;
    }
    .left-column, .right-column {
        width: 100%;
        padding: 20px;
    }
    .right-column {
        border-left: none;
        border-top:  none;
    }
}


.login-pf-page {
    margin-top: 0.5rem;
    font-size: 24px;
}


h1 {
	font-size: 32px;
	font-weight: 400;
}


.form-group {
    margin-bottom: 40px;
}


</style>
'
RuleException:
CalledProcessError in file "/users/omiralle/evalml/workflow/rules/inference.smk", line 23:
Command 'set -euo pipefail;  /users/omiralle/evalml/.venv/bin/python /users/omiralle/evalml/.snakemake/scripts/tmpj9wr51n9.set_inference_pyproject.py' returned non-zero exit status 1.
[Wed Aug 20 13:23:48 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 13
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/d0846032fc7248a58b089cbe8fa4c511/pyproject.toml
    log: output/logs/create_inference_pyproject/d0846032fc7248a58b089cbe8fa4c511.log (check log file(s) for error details)
Removing output files of failed job create_inference_pyproject since they might be corrupted:
output/data/runs/d0846032fc7248a58b089cbe8fa4c511/pyproject.toml
Traceback (most recent call last):
  File "/users/omiralle/evalml/.snakemake/scripts/tmpbsgyyaq5.set_inference_pyproject.py", line 419, in main
    subprocess.run(command, check=True, capture_output=True, text=True)
  File "/users/omiralle/.local/share/uv/python/cpython-3.12.7-linux-x86_64-gnu/lib/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['uv', '--project', '/scratch/mch/omiralle/evalenv/output/data/runs/9c18b90074214d769b8b383722fc5a06', 'add', '--no-sync', 'git+https://github.com/ecmwf/anemoi-inference@fix/interp_files']' returned non-zero exit status 1.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/omiralle/evalml/.snakemake/scripts/tmpbsgyyaq5.set_inference_pyproject.py", line 435, in <module>
    raise SystemExit(main(snakemake))
                     ^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.snakemake/scripts/tmpbsgyyaq5.set_inference_pyproject.py", line 426, in main
    raise RuntimeError("Failed to add extra dependencies") from e
RuntimeError: Failed to add extra dependencies
Traceback (most recent call last):
  File "/users/omiralle/evalml/.snakemake/scripts/tmpu_x2oiz7.set_inference_pyproject.py", line 419, in main
    subprocess.run(command, check=True, capture_output=True, text=True)
  File "/users/omiralle/.local/share/uv/python/cpython-3.12.7-linux-x86_64-gnu/lib/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['uv', '--project', '/scratch/mch/omiralle/evalenv/output/data/runs/07c3d9698db14d859b78bb712a65bbbf', 'add', '--no-sync', 'git+https://github.com/ecmwf/anemoi-inference@fix/interp_files']' returned non-zero exit status 1.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/omiralle/evalml/.snakemake/scripts/tmpu_x2oiz7.set_inference_pyproject.py", line 435, in <module>
    raise SystemExit(main(snakemake))
                     ^^^^^^^^^^^^^^^
  File "/users/omiralle/evalml/.snakemake/scripts/tmpu_x2oiz7.set_inference_pyproject.py", line 426, in main
    raise RuntimeError("Failed to add extra dependencies") from e
RuntimeError: Failed to add extra dependencies
RuleException:
CalledProcessError in file "/users/omiralle/evalml/workflow/rules/inference.smk", line 23:
Command 'set -euo pipefail;  /users/omiralle/evalml/.venv/bin/python /users/omiralle/evalml/.snakemake/scripts/tmpbsgyyaq5.set_inference_pyproject.py' returned non-zero exit status 1.
[Wed Aug 20 13:24:06 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 9
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/9c18b90074214d769b8b383722fc5a06/pyproject.toml
    log: output/logs/create_inference_pyproject/9c18b90074214d769b8b383722fc5a06.log (check log file(s) for error details)
Removing output files of failed job create_inference_pyproject since they might be corrupted:
output/data/runs/9c18b90074214d769b8b383722fc5a06/pyproject.toml
RuleException:
CalledProcessError in file "/users/omiralle/evalml/workflow/rules/inference.smk", line 23:
Command 'set -euo pipefail;  /users/omiralle/evalml/.venv/bin/python /users/omiralle/evalml/.snakemake/scripts/tmpu_x2oiz7.set_inference_pyproject.py' returned non-zero exit status 1.
[Wed Aug 20 13:24:06 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 26
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/07c3d9698db14d859b78bb712a65bbbf/pyproject.toml
    log: output/logs/create_inference_pyproject/07c3d9698db14d859b78bb712a65bbbf.log (check log file(s) for error details)
Removing output files of failed job create_inference_pyproject since they might be corrupted:
output/data/runs/07c3d9698db14d859b78bb712a65bbbf/pyproject.toml
[Wed Aug 20 13:24:06 2025]
Finished jobid: 4 (Rule: verif_metrics_cosmoe)
1 of 36 steps (3%) done
[Wed Aug 20 13:24:06 2025]
Finished jobid: 3 (Rule: verif_metrics_cosmoe)
2 of 36 steps (6%) done
Shutting down, this might take some time.
Cleaning up log files older than 10 day(s).
Exiting because a job execution failed. Look below for error messages
[Wed Aug 20 13:24:09 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 13
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/d0846032fc7248a58b089cbe8fa4c511/pyproject.toml
    log: output/logs/create_inference_pyproject/d0846032fc7248a58b089cbe8fa4c511.log (check log file(s) for error details)
[Wed Aug 20 13:24:09 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 9
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/9c18b90074214d769b8b383722fc5a06/pyproject.toml
    log: output/logs/create_inference_pyproject/9c18b90074214d769b8b383722fc5a06.log (check log file(s) for error details)
[Wed Aug 20 13:24:09 2025]
Error in rule create_inference_pyproject:
    message: None
    jobid: 26
    input: workflow/envs/anemoi_inference.toml
    output: output/data/runs/07c3d9698db14d859b78bb712a65bbbf/pyproject.toml
    log: output/logs/create_inference_pyproject/07c3d9698db14d859b78bb712a65bbbf.log (check log file(s) for error details)
Complete log(s): /users/omiralle/evalml/.snakemake/log/2025-08-20T132329.533702.snakemake.log
WorkflowError:
At least one job did not complete successfully.
